src/Tizen.Nlp/Interop/Interop.Nlp.cs

```diff
@@ -0,0 +1,306 @@
+            }
+
+            protected override void OnDisconnectedEvent(string endPoint, string portName)
+            {
+                _online = false;
+                Disconnected?.Invoke(this, null);
+            }
+
+            protected override void OnRejectedEvent(string endPoint, string portName)
+            {
+                Rejected?.Invoke(this, null);
+            }
+
+            private void ProcessReceivedEvent(Parcel parcel)
+            {
+                int id = parcel.ReadInt();
+                int seqId = parcel.ReadInt();
+                bool once = parcel.ReadBool();
+
+                foreach (var i in _delegateList)
+                {
+                    if ((int)i.Id == id && i.SeqId == seqId)
+                    {
+                        i.OnReceivedEvent(parcel);
+                        if (i.Once)
+                            _delegateList.Remove(i);
+                        break;
+                    }
+                }
+            }
+
+            protected override void OnReceivedEvent(string endPoint, string portName)
+            {
+                Parcel parcelReceived;
+
+                parcelReceived = new Parcel(CallbackPort);
+
+                using (parcelReceived)
+                {
+                    int cmd = parcelReceived.ReadInt();
+                    if (cmd != (int)MethodId.__Callback)
+                    {
+                        return;
+                    }
+
+                    ProcessReceivedEvent(parcelReceived);
+                }
+            }
+
+            private void ConsumeCommand(out Parcel parcel, Port port)
+            {
+                do
+                {
+                    var p = new Parcel(port);
+
+                    int cmd = p.ReadInt();
+                    if (cmd == (int)MethodId.__Result)
+                    {
+                        parcel = p;
+                        return;
+                    }
+
+                    p.Dispose();
+                    parcel = null;
+                } while (true);
+            }
+
+            public message(string appId) => _appId = appId;
+
+            /// <summary>
+            /// Connects to the service app.
+            /// </summary>
+            /// <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+            /// <privilege>http://tizen.org/privilege/datasharing</privilege>
+            /// <exception cref="InvalidIDException">
+            /// Thrown when the appid to connect is invalid.
+            /// </exception>
+            /// <exception cref="InvalidIOException">
+            /// Thrown when internal I/O error happen.
+            /// </exception>
+            /// <exception cref="PermissionDeniedException">
+            /// Thrown when the permission is denied.
+            /// </exception>
+            /// <remark> If you want to use this method, you must add privileges.</remark>
+            public void Connect()
```

Comment:
This method is not public because the message class is internal. you can remove the comment for documentation.